spec/turtle.bnf

```diff
@@ -23,10 +23,10 @@ String                ::= STRING_LITERAL_QUOTE | STRING_LITERAL_SINGLE_QUOTE
 iri                   ::= IRIREF | PrefixedName
 PrefixedName          ::= PNAME_LN | PNAME_NS
 BlankNode             ::= BLANK_NODE_LABEL | ANON
-quotedTriple          ::= '<<' qtSubject predicate qtObject '>>'
-qtSubject             ::=	iri | BlankNode | quotedTriple
-qtObject              ::=	iri | BlankNode | literal | quotedTriple
-annotation            ::= '{|' predicateObjectList '|}'
+reifier               ::= '<<' ((iri | BlankNode) '|' )? subject predicate object '>>'
```

Comment:
Since it is now well established what a `tripleTerm` is, why not take it from there and have `reifiedTerm` to describe what the `reifier` refers to in the baseline proposal. I employed the same logic to name `statedTerm` and `describedTerm` in my recent [proposal](https://lists.w3.org/Archives/Public/public-rdf-star-wg/2024Jul/0109.html) to disambiguate different kinds of tripleTerm instantiations.

tl;dr: `reifiedTerm`